packages/harpguru-core/src/components/covariant-menu/covariant-menu.tsx

```diff
@@ -87,27 +104,28 @@ export const CovariantMenu = ({
         },
       ]}
     >
-      <TapGestureHandler onHandlerStateChange={tapHandler}>
-        <Animated.View
-          style={[
-            styles.overlay,
-            {
-              backgroundColor: menuBackgroundColor,
-              opacity: menuOpacity,
-            },
-          ]}
-        >
-          <View style={styles.mainContents}>
-            <OptionLock locked={harpKeyIsLocked} handleTap={lockHarpKey}>
-              <Option {...harpKeyOptionProps} />
-            </OptionLock>
-            <OptionLock locked={pozitionIsLocked} handleTap={lockPozition}>
-              <Option {...pozitionOptionProps} />
-            </OptionLock>
-            <OptionLock locked={rootPitchIsLocked} handleTap={lockRootPitch}>
-              <Option {...rootPitchOptionProps} />
-            </OptionLock>
-          </View>
+      <Animated.View
+        style={[
+          styles.overlay,
+          {
+            backgroundColor: menuBackgroundColor,
+            opacity: menuOpacity,
+          },
+        ]}
+      >
+        <View style={styles.mainContents}>
+          <OptionLock locked={harpKeyIsLocked} handleTap={lockHarpKey}>
+            <Option {...harpKeyOptionProps} />
```

Comment:
It would probably be nice to know whether an option is locked in the option itself so that we can only present the selected option, so that the user doesn't try to select the other ones too.

We probably don't need the wrapping OptionLock anymore if we're passing the lock status to the Option itself. One of the main reasons for having this was to allow it to fit better in to the badly controlled vertical alignment which has been improved in this pull request anyway.

The `lock` property we would pass to the Option would probably need 3 values: 'locked', 'unlocked' and 'not-lockable' since the layout menu won't even have the locking functionality.

Reply:
Raised as tech debt in #20